### ISSUE TITLE
Add Position information for all countries

### ISSUE THREAD

**Issue by tmtmtmtm** [everypolitician/everypolitician#327]
DONE
- [x] Abkhazia/Assembly
- [x] Afghanistan/Wolesi_Jirga
- [x] Aland/Lagting
- [x] Albania/Assembly
- [x] American_Samoa/House
- [x] Andorra/General_Council
- [x] Antigua_and_Barbuda/Representatives
- [x] Argentina/Diputados
- [x] Armenia/Assembly
- [x] Australia/Representatives
- [x] Australia/Senate
- [x] Austria/Nationalrat
- [x] Azerbaijan/National_Assembly
- [x] Bahamas/House_of_Assembly
- [x] Bangladesh/House
- [x] Belarus/Chamber
- [x] Belgium/Representatives
- [x] Belize/Representatives
- [x] Bolivia/Deputies
- [x] Botswana/Assembly
- [x] Brazil/Deputies
- [x] British_Virgin_Islands/Assembly
- [x] British_Virgin_Islands/Council
- [x] Bulgaria/National_Assembly
- [x] Cambodia/National_Assembly
- [x] Canada/Commons
- [x] Chile/Deputies
- [x] Colombia/Representatives
- [x] Cook_Islands/Parliament
- [x] Costa_Rica/Assembly
- [x] Croatia/Sabor
- [x] Cyprus/House_of_Representatives
- [x] Czech_Republic/Deputies
- [x] Denmark/Folketing
- [x] Dominican_Republic/Diputados
- [x] Ecuador/Asamblea
- [x] Estonia/Riigikogu
- [x] Falkland_Islands/Assembly
- [x] Faroe_Islands/Logting
- [x] Fiji/Parliament
- [x] Finland/Eduskunta
- [x] France/National_Assembly
- [x] French_Polynesia/Assembly
- [x] Gabon/Assembly
- [x] Georgia/Parliament
- [x] Germany/Bundestag
- [x] Ghana/Parliament
- [x] Gibraltar/Parliament
- [x] Greece/Parliament
- [x] Greenland/Inatsisartut
- [x] Guam/Parliament
- [x] Guernsey/States
- [x] Honduras/Congreso_Nacional
- [x] Hong_Kong/Legislative_Council
- [x] Hungary/Assembly
- [x] Iceland/Assembly
- [x] India/Lok_Sabha
- [x] Indonesia/Council
- [x] Iran/Assembly
- [x] Ireland/Dail
- [x] Isle_of_Man/House_of_Keys
- [x] Israel/Knesset
- [x] Italy/House
- [x] Jamaica/House_of_Representatives
- [x] Japan/House_of_Representatives
- [x] Jordan/House_of_Representatives
- [x] Kenya/Assembly
- [x] Kiribati/Parliament
- [x] Kuwait/National_Assembly
- [x] Kyrgyzstan/Council
- [x] Latvia/Saeima
- [x] Lebanon/Parliament
- [x] Liechtenstein/Landtag
- [x] Lithuania/Seimas
- [x] Luxembourg/Chamber
- [x] Macao/Assembly
- [x] Macedonia/Sobranie
- [x] Madagascar/Assembly
- [x] Malaysia/Dewan_Rakyat
- [x] Malta/Assembly
- [x] Mexico/Deputies
- [x] Micronesia/Congress
- [x] Moldova/Parlamentul
- [x] Mongolia/Assembly
- [x] Montenegro/Assembly
- [x] Namibia/Assembly
- [x] Nauru/Parliament
- [x] Netherlands/House_of_Representatives
- [x] New_Caledonia/Congress
- [x] New_Zealand/House
- [x] Niue/Assembly
- [x] North_Korea/National_Assembly
- [x] Northern_Cyprus/Assembly
- [x] Northern_Ireland/Assembly
- [x] Northern_Mariana_Islands/House
- [x] Norway/Storting
- [x] Pakistan/Assembly
- [x] Peru/Congreso
- [x] Philippines/House
- [x] Poland/Sejm
- [x] Portugal/Assembly
- [x] Puerto_Rico/House_of_Representatives
- [x] Romania/Deputies
- [x] Russia/Duma
- [x] Saint_Barthelemy/Council
- [x] Saint_Kitts_and_Nevis/Assembly
- [x] Saint_Lucia/Assembly
- [x] Saint_Martin/Council
- [x] Saint_Pierre_and_Miquelon/Territorial_Council
- [x] Samoa/Parliament
- [x] San_Marino/Council
- [x] Scotland/Parliament
- [x] Senegal/Assembly
- [x] Serbia/National_Assembly
- [x] Singapore/Parliament
- [x] Slovakia/National_Council
- [x] Slovenia/National_Assembly
- [x] Solomon_Islands/Parliament
- [x] South_Africa/Assembly
- [x] South_Korea/National_Assembly
- [x] Spain/Congress
- [x] Sri_Lanka/Parliament
- [x] Sweden/Riksdag
- [x] Switzerland/National_Council
- [x] Taiwan/Legislative_Yuan
- [x] Tanzania/Assembly
- [x] Thailand/National_Legislative_Assembly
- [x] Togo/Assembly
- [x] Tunisia/Majlis
- [x] Turkey/Assembly
- [x] Turks_and_Caicos_Islands/Assembly
- [x] Tuvalu/Parliament
- [x] Uganda/Parliament
- [x] UK/Commons
- [x] Ukraine/Verkhovna_Rada
- [x] United_States_of_America/House
- [x] United_States_of_America/Senate
- [x] Uruguay/Deputies
- [x] Vanuatu/Parliament
- [x] Vatican_City/Pontifical_Commission
- [x] Venezuela/Assembly
- [x] Vietnam/National_Assembly
- [x] Wales/Assembly
- [x] Wallis_and_Futuna/Territorial_Assembly
- [x] Zambia/Assembly
- [x] Zimbabwe/Assembly

---

TODO
- [ ] Algeria/Majlis
- [ ] Argentina/Senado
- [ ] Cameroon/Assembly
- [ ] Colombia/Senate
- [ ] Congo-Kinshasa/Assembly
- [ ] Egypt/Parliament
- [ ] Italy/Senate
- [ ] Laos/Assembly
- [ ] Lesotho/Assembly
- [ ] Nepal/Assembly
- [ ] Palau/House_of_Delegates
- [ ] Papua_New_Guinea/Parliament
- [ ] Pitcairn/Island_Council
- [ ] Suriname/Assembly

---

No Wikidata
- [ ] Alderney/States
- [ ] Angola/National_Assembly
- [ ] Anguilla/Assembly
- [ ] Aruba/Estates
- [ ] Bahrain/Council_of_Representatives
- [ ] Barbados/House_of_Assembly
- [ ] Benin/National_Assembly
- [ ] Bermuda/Assembly
- [ ] Bhutan/Assembly
- [ ] Bosnia_and_Herzegovina/House_of_Representatives
- [ ] Brunei/Legislative_Council
- [ ] Burkina_Faso/Assembly
- [ ] Burundi/Assembly
- [ ] Cabo_Verde/Assembly
- [ ] Cameroon/Senate
- [ ] Cayman_Islands/Legislative_Assembly
- [ ] Chad/Assembly
- [ ] China/Congress
- [ ] Comoros/Assembly
- [ ] Congo-Brazzaville/Assembly
- [ ] Curacao/Estates
- [ ] Djibouti/Assembly
- [ ] Dominica/House_of_Assembly
- [ ] El_Salvador/Legislative_Assembly
- [ ] Gambia/National_Assembly
- [ ] Grenada/House_of_Representatives
- [ ] Guatemala/Congress
- [ ] Guinea-Bissau/Assembly
- [ ] Guyana/National_Assembly
- [ ] Haiti/Deputies
- [ ] Iraq/Majlis
- [ ] Ivory_Coast/Assembly
- [ ] Jersey/States
- [ ] Kazakhstan/Assembly
- [ ] Kosovo/Assembly
- [ ] Liberia/House
- [ ] Libya/House_of_Representatives
- [ ] Malawi/Assembly
- [ ] Maldives/Majlis
- [ ] Mali/Assembly
- [ ] Marshall_Islands/Nitijela
- [ ] Mauritania/National_Assembly
- [ ] Mauritius/National_Assembly
- [ ] Monaco/Council
- [ ] Montserrat/Assembly
- [ ] Morocco/House
- [ ] Mozambique/Assembly
- [ ] Myanmar/House_of_Representatives
- [ ] Nagorno_Karabakh/Assembly
- [ ] Namibia/Council
- [ ] Nicaragua/Asamblea
- [ ] Niger/Assembly
- [ ] Nigeria/Assembly
- [ ] Norfolk_Island/Assembly
- [ ] Oman/Majlis
- [ ] Panama/Assembly
- [ ] Paraguay/Deputies
- [ ] Rwanda/Deputies
- [ ] Saint_Helena/Legislative_Council
- [ ] Saint_Vincent_and_the_Grenadines/Assembly
- [ ] Sark/Chief_Pleas
- [ ] Saudi_Arabia/Shura
- [ ] Seychelles/Assembly
- [ ] Sierra_Leone/Parliament
- [ ] Sint_Maarten/Estates
- [ ] Somalia/Lower
- [ ] Somaliland/Representatives
- [ ] South_Ossetia/Parliament
- [ ] South_Sudan/Assembly
- [ ] Swaziland/Assembly
- [ ] Syria/Majlis
- [ ] Tajikistan/Representatives
- [ ] Timor_Leste/Parlamento
- [ ] Tonga/Assembly
- [ ] Transnistria/Supreme_Council
- [ ] Trinidad_and_Tobago/Representatives
- [ ] Trinidad_and_Tobago/Senate
- [ ] Turkmenistan/Mejlis
- [ ] United_Arab_Emirates/Federal_National_Council
- [ ] US_Virgin_Islands/Legislature
- [ ] Uzbekistan/Legislative_Chamber
- [ ] Yemen/Majlis
- [ ] Zimbabwe/Senate

**Comment by tmtmtmtm** [comment]
via

``` ruby
require 'json'
require 'colorize'
require 'pry'

# List which countries need a Wikidata Positions file

def json_from(json_file)
  JSON.parse(open(json_file).read, symbolize_names: true)
end

TYPES = [ "No Wikidata", "No reconciliation", "Multi-reconciliation", "TODO", "DONE" ]

def type(file)
  i = json_from(file)
  wikidata = i[:sources].find { |s| s[:type] == 'wikidata' } rescue binding.pry

  return 0 unless wikidata
  return 2 if wikidata[:merge].class == Array
  return 1 unless wikidata[:merge].key?(:reconciliation_file) 
  return 3 unless i[:sources].find{ |s| s[:type] == 'wikidata-positions' }
  return 4
end

Dir.glob("**/sources/instructions.json").group_by { |f| type(f) }.sort_by { |t, _| t }.reverse_each do |type, fs|
  puts TYPES[type]
  fs.each do |f|
    puts "* %s %s" % [ type == 4 ? "[x]" : "[ ]", f.split('/').reverse.drop(2).take(2).reverse.join("/") ]
  end
  puts "\n---\n"
end
```

**Comment by tmtmtmtm** [comment]
DONE
* [x] Abkhazia/Assembly
* [x] Afghanistan/Wolesi_Jirga
* [x] Aland/Lagting
* [x] Albania/Assembly
* [x] Algeria/Majlis
* [x] American_Samoa/House
* [x] Andorra/General_Council
* [x] Angola/National_Assembly
* [x] Antigua_and_Barbuda/Representatives
* [x] Argentina/Diputados
* [x] Argentina/Senado
* [x] Armenia/Assembly
* [x] Australia/Representatives
* [x] Australia/Senate
* [x] Austria/Nationalrat
* [x] Azerbaijan/National_Assembly
* [x] Bahamas/House_of_Assembly
* [x] Bangladesh/House
* [x] Belarus/Chamber
* [x] Belgium/Representatives
* [x] Belize/Representatives
* [x] Bermuda/Assembly
* [x] Bhutan/Assembly
* [x] Bolivia/Deputies
* [x] Botswana/Assembly
* [x] Brazil/Deputies
* [x] British_Virgin_Islands/Assembly
* [x] British_Virgin_Islands/Council
* [x] Bulgaria/National_Assembly
* [x] Cabo_Verde/Assembly
* [x] Cambodia/National_Assembly
* [x] Cameroon/Assembly
* [x] Canada/Commons
* [x] Chile/Deputies
* [x] Colombia/Representatives
* [x] Colombia/Senate
* [x] Congo-Kinshasa/Assembly
* [x] Cook_Islands/Parliament
* [x] Costa_Rica/Assembly
* [x] Croatia/Sabor
* [x] Cyprus/House_of_Representatives
* [x] Czech_Republic/Deputies
* [x] Denmark/Folketing
* [x] Dominican_Republic/Diputados
* [x] Ecuador/Asamblea
* [x] Egypt/Parliament
* [x] Estonia/Riigikogu
* [x] Falkland_Islands/Assembly
* [x] Faroe_Islands/Logting
* [x] Fiji/Parliament
* [x] Finland/Eduskunta
* [x] France/National_Assembly
* [x] French_Polynesia/Assembly
* [x] Gabon/Assembly
* [x] Georgia/Parliament
* [x] Germany/Bundestag
* [x] Ghana/Parliament
* [x] Gibraltar/Parliament
* [x] Greece/Parliament
* [x] Greenland/Inatsisartut
* [x] Guam/Parliament
* [x] Guernsey/States
* [x] Honduras/Congreso_Nacional
* [x] Hong_Kong/Legislative_Council
* [x] Hungary/Assembly
* [x] Iceland/Assembly
* [x] India/Lok_Sabha
* [x] Indonesia/Council
* [x] Iran/Assembly
* [x] Ireland/Dail
* [x] Isle_of_Man/House_of_Keys
* [x] Israel/Knesset
* [x] Italy/House
* [x] Italy/Senate
* [x] Jamaica/House_of_Representatives
* [x] Japan/House_of_Representatives
* [x] Jordan/House_of_Representatives
* [x] Kenya/Assembly
* [x] Kiribati/Parliament
* [x] Kosovo/Assembly
* [x] Kuwait/National_Assembly
* [x] Kyrgyzstan/Council
* [x] Laos/Assembly
* [x] Latvia/Saeima
* [x] Lebanon/Parliament
* [x] Lesotho/Assembly
* [x] Liechtenstein/Landtag
* [x] Lithuania/Seimas
* [x] Luxembourg/Chamber
* [x] Macao/Assembly
* [x] Macedonia/Sobranie
* [x] Madagascar/Assembly
* [x] Malaysia/Dewan_Rakyat
* [x] Maldives/Majlis
* [x] Mali/Assembly
* [x] Malta/Assembly
* [x] Mexico/Deputies
* [x] Micronesia/Congress
* [x] Moldova/Parlamentul
* [x] Mongolia/Assembly
* [x] Montenegro/Assembly
* [x] Morocco/House
* [x] Namibia/Assembly
* [x] Nauru/Parliament
* [x] Nepal/Assembly
* [x] Netherlands/House_of_Representatives
* [x] New_Caledonia/Congress
* [x] New_Zealand/House
* [x] Nigeria/Representatives
* [x] Nigeria/Senate
* [x] Niue/Assembly
* [x] North_Korea/National_Assembly
* [x] Northern_Cyprus/Assembly
* [x] Northern_Ireland/Assembly
* [x] Northern_Mariana_Islands/House
* [x] Norway/Storting
* [x] Oman/Majlis
* [x] Pakistan/Assembly
* [x] Palau/House_of_Delegates
* [x] Panama/Assembly
* [x] Papua_New_Guinea/Parliament
* [x] Peru/Congreso
* [x] Philippines/House
* [x] Pitcairn/Island_Council
* [x] Poland/Sejm
* [x] Portugal/Assembly
* [x] Puerto_Rico/House_of_Representatives
* [x] Romania/Deputies
* [x] Russia/Duma
* [x] Saint_Barthelemy/Council
* [x] Saint_Kitts_and_Nevis/Assembly
* [x] Saint_Lucia/Assembly
* [x] Saint_Martin/Council
* [x] Saint_Pierre_and_Miquelon/Territorial_Council
* [x] Samoa/Parliament
* [x] San_Marino/Council
* [x] Scotland/Parliament
* [x] Senegal/Assembly
* [x] Serbia/National_Assembly
* [x] Singapore/Parliament
* [x] Slovakia/National_Council
* [x] Slovenia/National_Assembly
* [x] Solomon_Islands/Parliament
* [x] Somalia/Lower
* [x] South_Africa/Assembly
* [x] South_Korea/National_Assembly
* [x] Spain/Congress
* [x] Sri_Lanka/Parliament
* [x] Suriname/Assembly
* [x] Sweden/Riksdag
* [x] Switzerland/National_Council
* [x] Taiwan/Legislative_Yuan
* [x] Tanzania/Assembly
* [x] Thailand/National_Legislative_Assembly
* [x] Togo/Assembly
* [x] Tunisia/Majlis
* [x] Turkey/Assembly
* [x] Turks_and_Caicos_Islands/Assembly
* [x] Tuvalu/Parliament
* [x] Uganda/Parliament
* [x] UK/Commons
* [x] Ukraine/Verkhovna_Rada
* [x] United_States_of_America/House
* [x] United_States_of_America/Senate
* [x] Uruguay/Deputies
* [x] US_Virgin_Islands/Legislature
* [x] Vanuatu/Parliament
* [x] Vatican_City/Pontifical_Commission
* [x] Venezuela/Assembly
* [x] Vietnam/National_Assembly
* [x] Wales/Assembly
* [x] Wallis_and_Futuna/Territorial_Assembly
* [x] Zambia/Assembly
* [x] Zimbabwe/Assembly

---
No Wikidata
* [ ] Alderney/States
* [ ] Anguilla/Assembly
* [ ] Aruba/Estates
* [ ] Bahrain/Council_of_Representatives
* [ ] Barbados/House_of_Assembly
* [ ] Benin/National_Assembly
* [ ] Bosnia_and_Herzegovina/House_of_Representatives
* [ ] Brunei/Legislative_Council
* [ ] Burkina_Faso/Assembly
* [ ] Burundi/Assembly
* [ ] Cameroon/Senate
* [ ] Cayman_Islands/Legislative_Assembly
* [ ] Chad/Assembly
* [ ] China/Congress
* [ ] Comoros/Assembly
* [ ] Congo-Brazzaville/Assembly
* [ ] Curacao/Estates
* [ ] Djibouti/Assembly
* [ ] Dominica/House_of_Assembly
* [ ] El_Salvador/Legislative_Assembly
* [ ] Gambia/National_Assembly
* [ ] Grenada/House_of_Representatives
* [ ] Guatemala/Congress
* [ ] Guinea-Bissau/Assembly
* [ ] Guyana/National_Assembly
* [ ] Haiti/Deputies
* [ ] Iraq/Majlis
* [ ] Ivory_Coast/Assembly
* [ ] Jersey/States
* [ ] Kazakhstan/Assembly
* [ ] Liberia/House
* [ ] Libya/House_of_Representatives
* [ ] Malawi/Assembly
* [ ] Marshall_Islands/Nitijela
* [ ] Mauritania/National_Assembly
* [ ] Mauritius/National_Assembly
* [ ] Monaco/Council
* [ ] Montserrat/Assembly
* [ ] Mozambique/Assembly
* [ ] Myanmar/House_of_Representatives
* [ ] Nagorno_Karabakh/Assembly
* [ ] Namibia/Council
* [ ] Nicaragua/Asamblea
* [ ] Niger/Assembly
* [ ] Norfolk_Island/Assembly
* [ ] Paraguay/Deputies
* [ ] Rwanda/Deputies
* [ ] Saint_Helena/Legislative_Council
* [ ] Saint_Vincent_and_the_Grenadines/Assembly
* [ ] Sark/Chief_Pleas
* [ ] Saudi_Arabia/Shura
* [ ] Seychelles/Assembly
* [ ] Sierra_Leone/Parliament
* [ ] Sint_Maarten/Estates
* [ ] Somaliland/Representatives
* [ ] South_Ossetia/Parliament
* [ ] South_Sudan/Assembly
* [ ] Swaziland/Assembly
* [ ] Syria/Majlis
* [ ] Tajikistan/Representatives
* [ ] Timor_Leste/Parlamento
* [ ] Tonga/Assembly
* [ ] Transnistria/Supreme_Council
* [ ] Trinidad_and_Tobago/Representatives
* [ ] Trinidad_and_Tobago/Senate
* [ ] Turkmenistan/Mejlis
* [ ] United_Arab_Emirates/Federal_National_Council
* [ ] Uzbekistan/Legislative_Chamber
* [ ] Yemen/Majlis
* [ ] Zimbabwe/Senate

---

**Comment by tmtmtmtm** [comment]
The remainder of these depend on #393

**Comment by tmtmtmtm** [comment]
superseded by #651
